cookbooks/overpass/recipes/default.rb

```diff
@@ -27,7 +27,7 @@
 basedir = data_bag_item("accounts", username)["home"]
 web_passwords = data_bag_item("web", "passwords")
 
-%w[bin site diffs db src munin].each do |dirname|
+%w[bin site diffs db src].each do |dirname|
```

Comment:
We probably want to delete the munin directory?

Reply:
Purged now.